Update service catalog tutorial for MI 4.4.0 (#1256)
wso2/docs-mi: en/docs/learn/integration-tutorials/service-catalog-tutorial.md

Now let's start designing the integration by adding the necessary artifacts.

#### Create HTTP Connection 

> 1. Go to **Project Explorer** and click on **+ Add Artifact**.

Reviewer: ```suggestion
1. Navigate to **Micro Integrator Project Explorer**.
2. Click on the **+** symbol to open the **Add Artifact** pane.
```

    <a href="{{base_path}}/assets/img/develop/create-artifacts/add-artifact-icon.png"><img src="{{base_path}}/assets/img/develop/create-artifacts/add-artifact-icon.png" alt="add artifact" width="80%"></a>

> 2. Go to **Connections** under **Other Artifacts** section. 

Reviewer: ```suggestion
3. Click **+ View More** under **Create an Integration**.
4. Select **Connections** under **Other Artifacts**. 
```

> 3. Click on **Http** and in the **Add New Connection** form, specify the following values to create the new HTTP connection.

Reviewer: ```suggestion
5. Select **Http**. 
6. In the **Add New Connection** form, specify the following values to create the new HTTP connection.
```

2. Select **Log** mediator in the **Mediators** section. The Log mediator logs messages when the request is received by the API resource. In this scenario, let's configure the Log mediator to display the following message: “Welcome to the HealthcareService”.

3. Once you select the Log mediator, the **Log** pane will be opened. Fill in the information in the table below:
<table>
    <tr>
         <th>Field</th>
         <th>Value</th>
         <th>Description</th>
    </tr>
    <tbody>
        <tr>
             <td>Log Category</td>
             <td><code>INFO</code></td>
             <td>Indicates that the log contains an informational message.</td>
        </tr>
        <tr>
             <td>Message</td>
             <td><code>Welcome to HealthcareService</code></td>
             <td>Indicates the message.</td>
        </tr> 
        <tr>
             <td>Description</td>
             <td><code>Request Log</code></td>
             <td>The <strong>Description</strong> field provides the name that appears for the Log mediator icon in the design view.</td>
        </tr>
    </tbody>
</table>

4. Click **Submit** to save the Log mediator configuration.

>     Let's configure a **Get** operation to send the request message to the `HealthcareService` endpoint and receive the response message.

Reviewer: ```suggestion
    Let's configure a `GET` operation to send the request message to the `HealthcareService` endpoint and receive the response message.
```

> 5. Click on the **+** icon in the sequence to add a **GET** operation for the **QueryDoctorConn** after the Log mediator. 

Reviewer: ```suggestion
5. Click on the **+** icon in the sequence to add a **GET** operation for **QueryDoctorConn** after the Log mediator. 
```

From the palette, select **Mediators** > **HTTP** > **GET** operation.

    <a href="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/http-connector-get-operation.png"><img src="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/http-connector-get-operation.png" alt="http connector get operation" width="30%"></a>

> 6. Once the **GET** operation form opened. Fill in the information in the table below:

Reviewer: ```suggestion
6. Once the **GET** operation form is opened, fill in the information in the table below:
```
